bots/lambda_with_admin_privileges.py

```diff
@@ -0,0 +1,60 @@
+"""
+## lambda_with_admin_privileges
+What it does: For each lambda it check all policy that grant blanket permissions ('*') to resources and
+             detach it from the lambda role
+Usage: AUTO: lambda_with_admin_privileges
+Note: The bot will detach the policies that have admin privileges from the lambda role so you will need to configure the specific
+     policies to grant positive permissions to specific AWS services or actions
+Limitations:None
+"""
+
+from botocore.exceptions import ClientError
+
+
+def run_action(boto_session, rule, entity, params):
+    text_output = ""
+
+    # Create an iam client
+    iam_client = boto_session.client('iam')
+
+    policy = entity["executionRole"]
+    role_name = policy["combinedPolicies"][0]["name"]
+    try:
+        arn_list = get_all_policies(policy)
+        print(arn_list)
+
+        for arn in arn_list:
+            result = iam_client.detach_role_policy(
```

Comment:
You can try and catch here in case one of the policy detach will fail you will keep trying all the others

Reply:
surround it with try and catch